docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -2040,7 +2041,7 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/bulkdata/core/maxPartitions`|number| The maximum number of simultaneous partitions that are processed per Export and Import |
 |`fhirServer/bulkdata/core/maxInputs`|number| The number of inputs allowed for $import |
 |`fhirServer/bulkdata/core/iamEndpoint`|string| Override the system's IAM endpoint |
-|`fhirServer/bulkdata/core/fastTxTimeout`|number| Time timeout for the fast implementations transaction |
+|`fhirServer/bulkdata/core/maxChunkReadTime`|string| Max time in milliseconds to read during a bulkdata export without type filters. The time should be three quarters of the transactionManager timeout (often the FHIR_TRANSACTION_MANAGER_TIMEOUT value). Note, this value is a string representation of a long value.|
```

Comment:
I think my preference would have been to put this in seconds and use a number rather than a string (similar to what we did on other config params).  However, I can definitely live with this design as well if you prefer to keep it in ms.

Reply:
I prefer the milliseconds use here.  FHIR_TRANSACTION_MANAGER_TIMEOUT can be set in fractional seconds.

Reply:
Right, I was thinking more like 90.5 seconds or 3.5 seconds.